build.sbt

```diff
@@ -41,7 +41,7 @@ resolvers +=
 
 graphqlCodegenStyle := Apollo
 graphqlCodegenJson := JsonCodec.Circe
-graphqlCodegenImports ++= List("java.util.UUID", "java.time.ZonedDateTime")
+graphqlCodegenImports ++= List("java.util.UUID", "java.time.ZonedDateTime", "java.time.LocalDateTime")
```

Comment:
Should all dates be handled as ZonedDateTime, and let the clients handle them as the need to? Though I think there is some issue with the client side last modified date?

Reply:
The problem is that we don't know and we don't need to know the timezone of the last modified date. We could get it from the browser but it's pretty meaningless. I've used LocalDateTime because otherwise we'd just have to choose a timezone.

Reply:
Fair enough